scalactic/src/main/scala/org/scalactic/Catcher.scala

```diff
@@ -122,6 +122,6 @@ object Catcher {
    *
    * @param partial the partial function that is used by returned extractor to determine matches
    */
-  def apply(partial: PartialFunction[Throwable, Boolean]): Catcher = new Catcher(partial)
+  def apply(partial: PartialFunction[Throwable, Boolean] | Null): Catcher = new Catcher(partial)
```

Comment:
**Nullable parameter**
Parameter that can be null and should therefore have a `| Null` type.